src/c76_menuCAL.hpp

```diff
@@ -1231,28 +1248,46 @@ void printCalibrationSubmenu()
 #if USE_GYRO_LEVEL == 1
   else if (calState == ROLL_OFFSET_CALIBRATION)
   {
+    calCount++;
     auto angles = Gyro::getCurrentAngles();
-    sprintf(scratchBuffer, "R: -------------");
-    makeIndicator(scratchBuffer, angles.rollAngle - mount.getRollCalibrationAngle());
-    lcdMenu.printMenu(scratchBuffer);
+    accumulator.rollAngle += angles.rollAngle;
+    if (calCount >= ACCUMULATOR_COUNT)
+    {
+      accumulator.rollAngle /= (float)ACCUMULATOR_COUNT;
+      sprintf(scratchBuffer, "R: ");
+      makeIndicator(scratchBuffer, accumulator.rollAngle - mount.getRollCalibrationAngle());
+      scratchBuffer[16] = 0;
```

Comment:
Use `'\0'` instead of `0` for null byte in string

Reply:
Fixed.